src/applications/hca/containers/IDPage.jsx

```diff
@@ -35,12 +47,16 @@ class IDPage extends React.Component {
               <p>
                 <strong>Want to skip this step?</strong>
               </p>
-              <button className="va-button-link" onClick={this.goToLogin}>
```

Comment:
In the previous version there was no `goToLogin` defined so this `onClick` did nothing